src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -94,3 +98,11 @@ void CalculatorProgrammerRadixOperators::IsErrorVisualState::set(bool value)
         NumberPad->IsErrorVisualState = m_isErrorVisualState;
     }
 }
+
+void CalculatorProgrammerRadixOperators::OnPropertyChanged(Object^ sender, PropertyChangedEventArgs^ e)
+{
+    if (e->PropertyName == CalculatorViewModelProperties::OpenParenthesisCount && closeParenthesisButton->FocusState != Windows::UI::Xaml::FocusState::Unfocused)
```

Comment:
You want to use `::FocusState::Unfocused` here.  The problem with `FocusState::Unfocused` is that `CalculatorProgrammerRadixOperators` is a `UserControl` so it has an inherited `FocusState` property. The compiler sees `FocusState::Unfocused` and thinks you are accessing that property, followed by `::Unfocused` which it doesn't understand.  The scope resolution operator, `::`, tells the compiler you are accessing the `FocusState` enumeration which exists in the global namespace after the `using Windows::UI::Xaml;` statement.
https://docs.microsoft.com/en-us/cpp/cpp/scope-resolution-operator


Same feedback for the other file.